licenses/third_party.md

```diff
@@ -74,3 +74,23 @@ please keep this up to date with every new library use.
 | gopkg.in/yaml.v2                     | Apache License 2 |
 | github.com/aws/aws-sdk-go            | Apache License 2 |
 | github.com/jmespath/go-jmespath      | Apache License 2 |
+
+
+# Javascript deps _used_ in production (exclude all test dependencies)
+
+| software                             | license          |
+| :----------                          | :------------:   |
+| @testing-library                     | MIT              |
```

Comment:
btw, we don't need to list *any* test dependency. only production dependencies. Effectively things that we ship to customers.